cmd/logview/main.go

```diff
@@ -0,0 +1,125 @@
+/*
+LICENSE
+  Copyright (C) 2025 the Australian Ocean Lab (AusOcean)
+
+  This is free software: you can redistribute it and/or modify it
+  under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  It is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  in gpl.txt. If not, see http://www.gnu.org/licenses/.
+*/
+
+// logview is a simple web server that streams netsender logs from a Raspberry Pi via SSH.
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"time"
+
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	username = "pi"
+	logFile  = "/var/log/netsender/netsender.log"
+)
+
+var raspberryPiAddr string
```

Comment:
These can be in a var block.

Also not sure this is worth fixing, but these don't really need to be globals. You could have them as fields in service or context struct, which has logHandler and StreamLogs as methods on. Then you just create a value of this struct in the main function.